content/en/docs/Installation/_index.md

```diff
@@ -35,6 +35,7 @@ A standard Kyverno installation consists of a number of different components, so
 * **Webhooks**
   * ValidatingWebhookConfigurations for receiving both policy and resource validation requests.
   * MutatingWebhookConfigurations for receiving both policy and resource mutating requests.
+      * Note: Webhook operations per resource is dynamically set if there are match/exclude operations mentioned in the policies applied. If for a resource no operations are set in match or exclude blocks, all default operations are applied in the webhooks rules.
```

Comment:
This is not for mutatingwebhook only right?

Reply:
Only validation? Is that what you mean?

Reply:
@anushkamittal2001 ?

Reply:
Can you clarify that only CREATE and UPDATE operations are currently configured for the mutating webhook?